webapp/portlet/src/main/webapp/vue-apps/popover/components/PopoverUser.vue

```diff
@@ -0,0 +1,182 @@
+<template>
+  <v-menu
+    v-model="menu"
+    rounded="rounded"
+    :close-on-content-click="false"
+    :left="popoverLeftPosition"
```

Comment:
the variable `popoverLeftPosition` seems not changed, why is it used here, is it for RTL ?